earlyoom.service

```diff
@@ -1,7 +1,7 @@
 [Service]
 StandardOutput=null
 StandardError=syslog
-ExecStart=/usr/local/bin/earlyoom
+ExecStart=${PREFIX}/bin/earlyoom
```

Comment:
An alternative to edit the unit file with sed would be really nice, though!

But I could not find a suitable feature to do some kind of replacement in plain `make`. :cry: